extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -352,6 +359,10 @@ export class AmpLiveList extends AMP.BaseElement {
 
     if (hasInsertItems) {
       promise = promise.then(() => {
+        if (this.isReverseOrder_ && this.itemsSlot_.lastElementChild) {
```

Comment:
nit: let's do one call to `this.viewport_.animateScrollIntoView` and move the logic to assignment of a `elementToScrollTo` variable

Reply:
done.